runelite-client/src/main/java/net/runelite/client/plugins/itemidentification/ItemIdentification.java

```diff
@@ -427,7 +427,130 @@
+
+	TOPAZ_RING(Type.JEWELLERY, "Topaz", "T", ItemID.TOPAZ_RING),
+	TOPAZ_NECKLACE(Type.JEWELLERY, "Topaz", "T", ItemID.TOPAZ_NECKLACE),
+	TOPAZ_BRACELET(Type.JEWELLERY, "Topaz", "T", ItemID.TOPAZ_BRACELET),
+	TOPAZ_AMULET_U(Type.JEWELLERY, "Topaz", "T", ItemID.TOPAZ_AMULET_U),
+	TOPAZ_AMULET(Type.JEWELLERY, "Topaz", "T", ItemID.TOPAZ_AMULET),
+
+	GOLD_RING(Type.JEWELLERY, "Gold", "GOL", ItemID.GOLD_RING),
+	GOLD_NECKLACE(Type.JEWELLERY, "Gold", "GOL", ItemID.GOLD_NECKLACE),
+	GOLD_BRACELET(Type.JEWELLERY, "Gold", "GOL", ItemID.GOLD_BRACELET),
+	GOLD_AMULET_U(Type.JEWELLERY, "Gold", "GOL", ItemID.GOLD_AMULET_U),
+	GOLD_AMULET(Type.JEWELLERY, "Gold", "GOL", ItemID.GOLD_AMULET),
+
+	SAPPHIRE_RING(Type.JEWELLERY, "Sapphir", "S", ItemID.SAPPHIRE_RING),
+	SAPPHIRE_NECKLACE(Type.JEWELLERY, "Sapphir", "S", ItemID.SAPPHIRE_NECKLACE),
+	SAPPHIRE_BRACELET(Type.JEWELLERY, "Sapphir", "S", ItemID.SAPPHIRE_BRACELET),
+	SAPPHIRE_AMULET_U(Type.JEWELLERY, "Sapphir", "S", ItemID.SAPPHIRE_AMULET_U),
+	SAPPHIRE_AMULET(Type.JEWELLERY, "Sapphir", "S", ItemID.SAPPHIRE_AMULET),
+
+	EMERALD_RING(Type.JEWELLERY, "Emerald", "E", ItemID.EMERALD_RING),
+	EMERALD_NECKLACE(Type.JEWELLERY, "Emerald", "E", ItemID.EMERALD_NECKLACE),
+	EMERALD_BRACELET(Type.JEWELLERY, "Emerald", "E", ItemID.EMERALD_BRACELET),
+	EMERALD_AMULET_U(Type.JEWELLERY, "Emerald", "E", ItemID.EMERALD_AMULET_U),
+	EMERALD_AMULET(Type.JEWELLERY, "Emerald", "E", ItemID.EMERALD_AMULET),
+
+	RUBY_RING(Type.JEWELLERY, "Ruby", "R", ItemID.RUBY_RING),
+	RUBY_NECKLACE(Type.JEWELLERY, "Ruby", "R", ItemID.RUBY_NECKLACE),
+	RUBY_BRACELET(Type.JEWELLERY, "Ruby", "R", ItemID.RUBY_BRACELET),
+	RUBY_AMULET_U(Type.JEWELLERY, "Ruby", "R", ItemID.RUBY_AMULET_U),
+	RUBY_AMULET(Type.JEWELLERY, "Ruby", "R", ItemID.RUBY_AMULET),
+
+	DIAMOND_RING(Type.JEWELLERY, "Diamon", "DI", ItemID.DIAMOND_RING),
+	DIAMOND_NECKLACE(Type.JEWELLERY, "Diamon", "DI", ItemID.DIAMOND_NECKLACE),
+	DIAMOND_BRACELET(Type.JEWELLERY, "Diamon", "DI", ItemID.DIAMOND_BRACELET),
+	DIAMOND_AMULET_U(Type.JEWELLERY, "Diamon", "DI", ItemID.DIAMOND_AMULET_U),
+	DIAMOND_AMULET(Type.JEWELLERY, "Diamon", "DI", ItemID.DIAMOND_AMULET),
+
+	DRAGONSTONE_RING(Type.JEWELLERY, "Dragon", "DR", ItemID.DRAGONSTONE_RING),
+	DRAGON_NECKLACE(Type.JEWELLERY, "Dragon", "DR", ItemID.DRAGON_NECKLACE),
+	DRAGONSTONE_BRACELET(Type.JEWELLERY, "Dragon", "DR", ItemID.DRAGONSTONE_BRACELET),
+	DRAGONSTONE_AMULET_U(Type.JEWELLERY, "Dragon", "DR", ItemID.DRAGONSTONE_AMULET_U),
+	DRAGONSTONE_AMULET(Type.JEWELLERY, "Dragon", "DR", ItemID.DRAGONSTONE_AMULET),
+
+	ONYX_RING(Type.JEWELLERY, "Onyx", "ON", ItemID.ONYX_RING),
+	ONYX_NECKLACE(Type.JEWELLERY, "Onyx", "ON", ItemID.ONYX_NECKLACE),
+	ONYX_BRACELET(Type.JEWELLERY, "Onyx", "ON", ItemID.ONYX_BRACELET),
+	ONYX_AMULET_U(Type.JEWELLERY, "Onyx", "ON", ItemID.ONYX_AMULET_U),
+	ONYX_AMULET(Type.JEWELLERY, "Onyx", "ON", ItemID.ONYX_AMULET),
+
+	ZENYTE_RING(Type.JEWELLERY, "Onyx", "ON", ItemID.ZENYTE_RING),
```

Comment:
medName and shortName are wrong for this group

Reply:
My bad, fixed it just now